architecture/core_concepts/routes.adoc

```diff
@@ -171,34 +171,9 @@ no-route-hostname.my-namespace.router.default.svc.cluster.local <1>
 *router.default.svc.cluster.local*.
 ====
 
-=== Custom Default Routing Subdomain
-A cluster administrator can customize the suffix or the default routing
-subdomain for an environment using the
-link:../../admin_guide/master_node_configuration.html#master-configuration-files[master
-configuration file] (the *_/etc/openshift/master/master-config.yaml_* file by
-default). The following example shows how you can set the configured suffix to
-*v3.openshift.test*:
-
-.Master Configuration Snippet
-====
-
-----
-routingConfig:
-  subdomain: v3.openshift.test
-----
-====
-
-With the OpenShift master(s) running the above configuration, the
-generated host name for the example of a host added to a namespace
-*my-namespace* would be:
-
-.Generated Host Name
-====
-
-----
-no-route-hostname.my-namespace.v3.openshift.test
-----
-====
+A cluster administrator can also
+link:../../admin_guide/install/deploy_router.html#customizing-the-default-routing-subdomain[customize
+the suffix or the default routing subdomain] for their environment.
```

Comment:
"or" seems ambiguous, how about:

    customize the suffix used as the default routing subdomain